src/containers/shared/components/Transaction/NFTokenBurn/types.ts

```diff
@@ -0,0 +1,10 @@
+import { TransactionCommonFields } from '../types'
+
+export interface NFTokenBurn extends TransactionCommonFields {
+  NFTokenID: string
+  Owner?: string
```

Comment:
The `Owner` is never displayed anywhere, is that intentional?

Reply:
Probably not.  We should add that to the list of missed fields.

Reply:
I added it because NFToken has Owner? in XRPL.org description. However, it doesn't seem like we use it anywhere. Does adding it to list of missed fields mean that we might display it on Explorer page in future? Also, should I keep the Owner in for this PR?

Reply:
Keeping it is fine because it is correct.  We should create a ticket to add the Owner field to the simple view

Reply:
Created #356